include/TuningsImpl.h

```diff
@@ -928,5 +935,215 @@ inline KeyboardMapping startScaleOnAndTuneNoteTo(int scaleStart, int midiNote, d
     return parseKBMData(oss.str());
 }
 
+inline AbletonScale readASCLStream(std::istream &inf)
+{
+    AbletonScale as;
+
+    /**
+     * Reverse engineering Ableton Tuning web app to understand ASCL to KBM export.
+     *
+     * @see https://tuning.ableton.com/squigadooServer/squigadoo/modular-playground
+     *
+     * ```
+     * scalaKbmFile: {
+     *   filename: r,
+     *   text: (0, a.F)(i, 0, 127, C(0, t), C(t.referencePitchIndex, t), c.referencePitchFrequency, i, f)
+     * }
+     * ```
```

Comment:
This code here - is it copied from somewhere which allows redistribution in an MIT licensed package?

Reply:
I lifted this code from the Ableton Tuning website to reverse-engineer their method of converting ASCL to KBM. I will remove these comments when I submit my PR.